src/Perpetuum/Zones/NpcSystem/Npc.cs

```diff
@@ -168,6 +168,11 @@ private bool IsAttackable(Hostile hostile)
             if (hostile.unit.IsInvulnerable)
                 return false;
 
+            if (hostile.unit is Player)
+            {
+                return (hostile.unit as Player).GetLocks().OfType<UnitLock>().Where(x => x.Target.Eid == npc.Eid).Any();
```

Comment:
Use a static cast instead of the safety cast. Better performance.

Reply:
Question: is 
```
            Player player = hostile.unit as Player;
            if (player != null)
            {
                return player.GetLocks().OfType<UnitLock>().Where(x => x.Target.Eid == npc.Eid).Any();
            }
```
better than:
```
            if (hostile.unit is Player player)
            {
                return player.GetLocks().OfType<UnitLock>().Where(x => x.Target.Eid == npc.Eid).Any();
            }
```
?

I see that `as` more efficiently evaluates type or fallsback to null (which requires that null-check), but is that the same as the alternative shown above, given that we have to null-check on that statement?